packages/webapp/src/components/Animals/FixedHeaderContainer/index.tsx

```diff
@@ -22,12 +22,17 @@ import React, {
   useRef,
   useState,
 } from 'react';
+import { matchPath } from 'react-router-dom';
 import clsx from 'clsx';
 import { Paper } from '@mui/material';
+import history from '../../../history';
+import { ADD_ANIMALS_URL } from '../../../util/siteMapConstants';
 import styles from './styles.module.scss';
 
 const PAPER_BORDER = 2;
 
+const FULL_WIDTH_ROUTES = [ADD_ANIMALS_URL];
```

Comment:
It would be a few more strings for the `.some()` to check (and that might be enough reason), but is there another downside to exporting the App `FULL_WIDTH_ROUTES` into this component so we are defining those routes in just one place instead of two?

Reply:
I initially thought about this too but realized that these `FULL_WIDTH_ROUTES` serve two different purposes.
For example, the inventory page needs to be full width in the App because of the KPI, but the content itself still needs to be max-width of 1024px. (it will be cumbersome when needing to add a route to both files though...)

Maybe renaming it to `CONTAINER_FULL_WIDTH_ROUTES` could be less confusing? Or passing a CSS class to `FixedHeaderContainer` might be better??

Reply:
Ohhhh, I see! In that case, yes maybe a different constant name because the identical naming made me automatically think of the other one and assume the function would be the same too.

Can you tell me more about passing a CSS class? I might have thought just a prop `isFullWidth` or `fullWidthContent` would work too, but in both cases this would have to be passed all the way through e.g. MultiStepPageForm, is that right? Since some forms will I suppose be full-width and others not?

Reply:
Passing a CSS class would be similar to passing `isFullWidth` or `fullWidthContent` - instead of a boolean, we would pass something like `{ classes.paper: <className that has max-width: 100%> }`. But you are right, it wouldn't be an enjoyable approach! I'll rename the constant instead. Thank you! ❤️